emmaa/aws_lambda_functions/update_lambda.py

```diff
@@ -1,31 +1,40 @@
 import boto3
+import sys
 from os import path
 from zipfile import ZipFile
 
 HERE = path.dirname(path.abspath(__file__))
+script_name = sys.argv[1]
+function_name = sys.argv[2]
 
-
-def upload_function():
+def upload_function(script_name, function_name):
     """Upload the lambda function by pushing a zip file to Lambda.
 
     This function pre-supposes you are running from the same directory that
-    contains the lambda script, which should be named: `script.py`.
+    contains the lambda script. 
+    
+    Parameters
+    ----------
+    script_name : str
+        Name of a script containing lambda function. 
+    function_name : object
+        Name of a lambda function as specified on AWS Lambda.
     """
     lamb = boto3.client('lambda')
     with ZipFile(path.join(HERE, 'lambda.zip'), 'w') as zf:
-        zf.write(path.join(HERE, 'script.py'),
-                 'emmaa/%s/script.py' % path.basename(HERE))
+        zf.write(path.join(HERE, script_name),
+                 'emmaa/{0}/{1}'.format(path.basename(HERE), script_name))
```

Comment:
Though in no way critical, you can actually use f-strings here to make this prettier: e.g. `f'emma/{path.basename(HERE)}/{script_name}'`.